systems/citybuilding.go

```diff
@@ -46,29 +53,49 @@ func (cb *CityBuildingSystem) New(w *ecs.World) {
 // Update is ran every frame, with `dt` being the time
 // in seconds since the last frame
 func (cb *CityBuildingSystem) Update(dt float32) {
-	if engo.Keys.Get(engo.F1).JustPressed() {
-		fmt.Println("The gamer pressed F1")
+	keyQ := engo.Button{[]engo.Key{engo.Q}, "Q"}
```

Comment:
It's customary (for as far as we have customs) to, instead of writing `"Q"`, using something semantic / meaningful. Like this:

```go
engo.Button{[]engo.Key{engo.Q}, "build"}
```

That way, regardless of which actual physical key triggers that one (maybe the gamer can set custom key bindings? ) the semantics still stay the same. You also won't have to remember which keys do what (when checking the value of it), since you can simply ask "hmm, how is the "build" key doing? "